crates/sui-graphql-client/src/lib.rs

```diff
@@ -235,22 +235,22 @@ impl DynamicFieldOutput {
     pub fn deserialize_name<T: DeserializeOwned>(
         &self,
         expected_type: &TypeTag,
-    ) -> Result<T, anyhow::Error> {
+    ) -> Result<T, error::Error> {
         assert_eq!(
             expected_type, &self.name.type_,
             "Expected type {}, but got {}",
             expected_type, &self.name.type_
         );
 
         let bcs = &self.name.bcs;
-        bcs::from_bytes::<T>(bcs).map_err(|_| anyhow!("Cannot decode BCS bytes"))
+        bcs::from_bytes::<T>(bcs).map_err(|e| e.into())
```

Comment:
For these you should be able to do 
```suggestion
        bcs::from_bytes::<T>(bcs).map_err(Into::into)
```